node/worker/worker.go

```diff
@@ -328,6 +331,21 @@ func (w *Worker) FinalizeNewBlock(
 	w.current.header.SetCoinbase(coinbase)
 	w.current.header.SetViewID(new(big.Int).SetUint64(viewID))
 
+	// Slashes
```

Comment:
We may need to slash later (not immediately) so as to wait for all the double sign on the same block to appear. Probably at the end of epoch is better.